.github/workflows/skywalking.yaml

```
@@ -22,7 +22,7 @@ on:
     - cron: "0 18 * * *" # TimeZone: UTC 0
 
 concurrency:
-  group: skywalking-${{ github.event.pull_request.number || github.ref }}
+  group: skywalking-graalvm-${{ github.event.pull_request.number || github.ref }}
```

I think you changed the wrong place, this file is `skywalking.yaml`, you should change the one in file `.github/workflows/native-image.yaml`

```suggestion
  group: skywalking-${{ github.event.pull_request.number || github.ref }}
```